libs/sdk-ui/src/base/index.tsx

```diff
@@ -182,7 +182,7 @@ export {
     pickCorrectInsightWording,
     pickCorrectMetricWording,
     pickCorrectWording,
-    removeAllInsightToReportTranslations,
+    removeAllWordingTranslationsWithSpecialSuffix,
```

Comment:
I think we shouldn't remove `removeAllInsightToReportTranslations` now, as we are working with the sdk and the function is still probably used in some apps. Removing something without marking it as `deprecated` for a while might cause trouble to those who are using the function when they bump the sdk version in their apps...